src/lib/ddl/shopping-ddl.js

```diff
@@ -0,0 +1,42 @@
+/* @flow */
+import { createEventHandler } from './event-handlers';
+
+const snippetRunEvent = { event: 'snippetRun', t: new Date().getTime() };
+export const DDL_NAME = 'shoppingDDL';
+
+const _getDDL = (w) => {
```

Comment:
`w` what does it mean?

Reply:
it is a window object. However, we do not want it to explicitly name it as it is a global object

Reply:
Why do we need it if it is global object?